internal/services/dynatrace/dynatrace_monitor_resource.go

```diff
@@ -32,6 +32,7 @@ type MonitorsResourceModel struct {
 	MonitoringStatus              bool                           `tfschema:"monitoring_enabled"`
 	MarketplaceSubscriptionStatus string                         `tfschema:"marketplace_subscription"`
 	Identity                      []identity.ModelSystemAssigned `tfschema:"identity"`
+	EnvironmentProperties         []EnvironmentProperties        `tfschema:"environment_properties"`
```

Comment:
could we rename this to just `environment`?

```suggestion
	Environment       []Environment     `tfschema:"environment"`
```

Reply:
Sure. Will do.